vendor/brave-ios/Ledger/BATBraveLedger.mm

```diff
@@ -1503,4 +1496,19 @@ - (void)onContributeUnverifiedPublishers:(ledger::Result)result publisherKey:(co
   }
 }
 
+- (void)getExternalWallets:(ledger::GetExternalWalletsCallback)callback
+{
+
```

Comment:
Are there any implications of not calling the callback? If so, perhaps we should call it with a failure for now?